Documentation/DocuBlocks/Rest/Indexes/post_api_index_inverted.md

```diff
@@ -95,18 +95,17 @@ An attribute path. The `.` character denotes sub-attributes.
 
 @RESTSTRUCT{analyzer,post_api_index_inverted_nested,string,optional,}
 The name of an Analyzer to use for this field.
-Default: the value defined by the top-level `analyzer` option.
+Default: the value defined by the parent field, or the top-level `analyzer` option.
 
 @RESTSTRUCT{features,post_api_index_inverted_nested,array,optional,string}
-A list of Analyzer features to use for this field. They define what features are
-enabled for the `analyzer`. Possible features:
+A list of Analyzer features to use for this field. You can set this option to
+overwrite what features are enabled for the `analyzer`. Possible features:
 - `"frequency"`
 - `"norm"`
 - `"position"`
 - `"offset"`
 
-Default: the value of the top-level `features` option, or if not set, the
-features defined by the Analyzer itself.
+Default: the features as defined by the Analyzer itself.
```

Comment:
```suggestion
Default: the features as defined by the Analyzer itself, or by parent features field in case of analyzer is not set for this field
```

Reply:
features are inherited from parent/top-level unless analyzer is set further down